utils/chem/chem_diagb.h

```diff
@@ -230,15 +236,66 @@ namespace gdasapp {
         }
       }
 
+      std::cout << "Number of variables in chemVars: " << chemVars.size() << std::endl;
+
+      for (size_t i = 0; i < chemVars.size(); ++i) {
+        std::cout << "Variable " << i << ": " << chemVars[i].name() << std::endl;
+}
+
       // Rescale
+      //
       if (fullConfig.has("rescale")) {
-        double rescale;
-        fullConfig.get("rescale", rescale);
-        util::multiplyFieldSet(bkgErrFs, rescale);
```

Comment:
What about using `multiplyFieldSets`? I think this would require creating a 3D array of values in a new field set, but that would then set us up for the ability to scale by height/latitude/location at a later date. Thoughts?

Reply:
I can look at it and see how that would work. It took me some time to make something that could run....

Reply:
I did try multiplyFieldSets, but had trouble getting it to compile.

Reply:
I wonder if we just read in another 'state' that is the scaling factors and be proactive on the 3D front?

Reply:
So have a file with the scaling factors, and read it in from there?

Reply:
I think so, that way we can easily scale by lat,lon,height and species

Reply:
Oh, I think I understand. A full background state file, filled with rescaling values for each aerosol species and at all locations.   I'd like to try just 14 values first and see if we can do better.

Reply:
yeah they can be constant in x,y,z to start